src/SignalR/clients/ts/FunctionalTests/SignalR.Npm.FunctionalTests.npmproj

```diff
@@ -31,7 +31,7 @@
     <!-- Skip the "inner" test run when we're running DailyTests -->
     <Yarn Command="run test:inner --no-color --configuration $(Configuration)"
           Condition="'$(DailyTests)' != 'true'"
-          WorkingDirectory="$(RepoRoot)src/SignalR/clients/ts/FunctionalTests" />
+          WorkingDirectory="$(MSBuildThisFileDirectory)" />
```

Comment:
This was a teensy nit. No reason to describe the path in so much detail when `$(MSBuildThisFileDirectory)` is sitting right there&hellip;